mass_mailing_event/models/event_state.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class EventState(models.Model):
+    _name = 'event.state'
+
+    name = fields.Char(required=True)
```

Comment:
Thanks for your explanation but I think this is simpler and event registration states are not expected to change. And in that case, admin could add the new state to the table via XML.

I'll change model name, thanks.